docs/reference/data-streams/lifecycle/index.asciidoc

```diff
@@ -2,7 +2,7 @@
 [[data-stream-lifecycle]]
 == Data stream lifecycle
 
-A data stream lifecycle is the built-in mechanism data streams use to manage their lifecycle. It enables you to easily
+A data stream lifecycle is the built-in mechanism <<data-streams,data streams>> use to manage their <<index-lifecycle-management,lifecycle>>. It enables you to easily
```

Comment:
this link is not relevant here because this is different tech.

```suggestion
A data stream lifecycle is the built-in mechanism data streams use to manage their lifecycle. It enables you to easily
```